### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by Rustamxon7**
**Bookstore: initialize the project with components**

All requirements are completed: 

- React App is initialized 💯 
- React Redux added 💯 
- Used ducks pattern for Redux files 💯 

Feel free to review -🧑🏻‍💻 - my project and let me know if there are any changes to do. -🤓 -

### CODE REVIEW

File: src/components/bookstore/Book.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Book = ({
+  id, title, author, categories,
+}) => (
+  <li style={{ width: '50rem' }} key={id} id={id}>
+    <p style={{ width: '10rem' }}>{title}</p>
```
Comment (EricMbouwe) [inline src/components/bookstore/Book.js:8]: - [ ] You don't need to add the `key` attribute here, It is needed where you have used the map function, Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity. You can learn more about it [here](https://reactjs.org/docs/lists-and-keys.html)

Here is an example
```
const todoItems = todos.map((todo) =>
  <li key={todo.id}>
    {todo.text}
  </li>
```

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Rustamxon7,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/bookstore/Book.js (hunk 1 above)
Comment (Rustamxon7) [reply src/components/bookstore/Book.js:8]: Thank you again for reminding me about that.
And take a look at the solutions for this issues and let me know if I did correct. If I couldn't do as you requested, I would be very happy for explaining me the problem again. You can consider me as a new student for React&Redux.... 
**_Happy Reviewing!_**

File: src/components/bookstore/List.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      author: 'Dan Brown',
+      categories: 'adventure',
+    },
+  ];
+  return (
+    <ul>
+      {booksStore.map((book) => (
+        <Book key={book.id} title={book.title} author={book.author} categories={book.categories} />
+      ))}
+    </ul>
```
[the remark this answers is not in the document]
Comment (Rustamxon7) [reply src/components/bookstore/List.js:30]: This issue is also being complete!

File: src/components/bookstore/Input.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Input = () => (
+  <form>
+    <input type="text" placeholder="Book Title" />
+    <input type="text" placeholder="Categories" />
+    <button type="button">Add Book</button>
+  </form>
```
[the remark this answers is not in the document]
Comment (Rustamxon7) [reply src/components/bookstore/Input.js:8]: Thank you so much reminding me! I didn't pay attention for that. But this time I hope it is done!

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**